app/src/main/java/com/example/sabonit/Account.java

```diff
@@ -98,20 +105,20 @@ public void setCart(Cart cart)
     }
 
     /**
-     * Returns the UID of the account.
+     * Returns the uid of the account.
      */
-    public String getUID()
+    public String getUid()
     {
-        return UID;
+        return uid;
     }
 
     /**
      * Sets the confirmed orders of the account.
-     * @param UID given UID.
+     * @param uid given uid.
      */
-    public void setUID(String UID)
+    public void setUid(String uid)
     {
-        this.UID = UID;
+        this.uid = uid;
     }
 
```

Comment:
You defined good style,  and specifiec this change, that account write to the db is exactly what we talked about, great!